docfx_project/articles/bootstrapping-trust/part-01.encryption-everywhere.md

```diff
@@ -0,0 +1,207 @@
+# Bootstrapping Trust
+
+## Part 1: Encryption Everywhere
+
+### Zero Trust
+
+Bootstrapping Trust is a series highlighting a common problem dealing
+with software systems that span multiple machines over any network. All
+networks, private (like your office/data center) or public (like the
+internet), should be considered insecure. Firewalls and segmented
+networks are not enough. Even application specific networking is not
+enough. The next step is to not trust the network.
+
+This thought process is a new way of conceptualizing network security
+and is referred to as "Zero Trust." Not that trust is not present, far
+from it. Today's networks are the home to distributed software
+solutions. Whether they are high scale mesh networks or small scale
+client servers: there are distributed software running everywhere.
+Distributed systems are rife with trust and trust issues. Zero Trust
+means: "Zero Trust Without Verification." And it turns out that putting
+that trust in pace isn't simple.
+
+Whether you are an encryption expert or a newcomer, welcome! This series
```

Comment:
three paragraphs in and I'm now feeling the love of being welcomed. :) Personally I think this section could be stuffed way up at the beginning ahead of the Zero Trust definition even. This sets the stage for what comes next and seems like a really good lead in to the paragraphs ahead of this one

Reply:
Good call, moved to the top.